renovate.json

```diff
@@ -16,7 +16,8 @@
     "group:monorepos",
     "group:recommended",
     "helpers:disableTypesNodeMajor",
-    "helpers:oddIsUnstablePackages"
+    "helpers:oddIsUnstablePackages",
+    "schedule:nonOfficeHours"
```

Comment:
Is it clever enough to know which timezone?

Reply:
Good question, I assumed it'd be fine since this preset is being used in Page kit but I'll check

Reply:
> By default, Renovate schedules will use the timezone of the machine that it's running on.

I'm going to set it default to UK, do we have teams that use renovate in the offices outside the UK?

Reply:
Looks like in practice it uses UTC, but will set regardless.

> 📅 Schedule: "after 10pm every weekday,before 5am every weekday,every weekend" (UTC).

From https://github.com/Financial-Times/dotcom-page-kit/pull/814

Reply:
Interesting etymology about UTC:

> English speakers originally proposed CUT (for "coordinated universal time"), while French speakers proposed TUC (for "temps universel coordonné"). The compromise that emerged was UTC,[7] which conforms to the pattern for the abbreviations of the variants of Universal Time (UT0, UT1, UT2, UT1R, etc.).

Reference: https://en.wikipedia.org/wiki/Coordinated_Universal_Time.

Reply:
> I'm going to set it default to UK, do we have teams that use renovate in the offices outside the UK?

There's always a possibility that Ops Support in Manila might try and deploy a change if there is an out-of-hours emergency, but I don't think these are too often.

I am also trying to find evidence of apps used by FT Core (Sofia, Bulgaria) using the same CircleCI account as Customer Products.

Reply:
Yup, seems there are certainly some repos used by the Sofia-based team that share our CircleCI: https://github.com/Financial-Times/specialist-session.

Not sure how many in total.

Reply:
We could add some documentation explaining how to override the default and send it to that team? 🤔

Reply:
I think Sofia is a couple hours a head, so given the times of 5am and 10pm, this should be flexible enough for them too :)